Nosenti/certificate-manager---Innocent
# Task5-KAN-36 Deleting Certificate

#### PR description
- Pull request to delete a given certificate

#### Ticket / Jira Link
- [KAN-36](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?assignee=712020%3A67b4d316-bd6f-4b16-a625-69afa0078a10&selectedIssue=KAN-36)

#### Checklist:
- [x] I have produced the screenshots to show before and after delete action

#### How To Test Changes
- Click on Cog Icon in the Certificates Table
- Click on the delete option and the row of certificate chosen will be selected.
- If you refresh, It should still not be there because it is deleted in IndexDB

#### Screenshots (Before and After deleting the first row):
![image](https://github.com/user-attachments/assets/a0535c84-c3d0-42a6-8967-37f157549ae7)
![image](https://github.com/user-attachments/assets/102d1951-cffe-47f4-8fb8-4b667ed94f69)
